documentation/Using-the-SDK/Using-the-engine.md

```diff
@@ -118,6 +121,26 @@ using (var dataSources = DataSourceSet.Create(plugins))
 }
 ````
 
+If you would like to enable a composite cooker, call the `GetCookedData(DataCookerPath)`
```

Comment:
It's slightly incorrect to say you need to do this to "enable" a composite cooker. The composite cooker is enabled the same way any other cooker is enabled. I think instead we could have a "NOTE" here. Something like

```md
> ⚠️ Note: [Composite cookers](../Glossary.md#compositedatacooker) are processed lazily. The above code will not cause any code inside of an enabled composite cooker to execute. To execute/debug composite cookers, you must also have code that queries for the specific cooker, such as building a table that uses the cooker or manually calling `results.GetCookedData(CompCookerPath)` after processing.
```

Which would look like

> ⚠️ Note: [Composite cookers](../Glossary.md#compositedatacooker) are processed lazily. The above code will not cause any code inside of an enabled composite cooker to execute. To execute/debug composite cookers, you must also have code that queries for the specific cooker, such as building a table that uses the cooker or manually calling `results.GetCookedData(CompCookerPath)` after processing.